ci-operator/step-registry/hypershift/kubevirt/install/hypershift-kubevirt-install-commands.sh

```diff
@@ -101,7 +89,7 @@ spec:
   channel: ${CNV_RELEASE_CHANNEL}
   installPlanApproval: Automatic
   name: kubevirt-hyperconverged
-  source: ${CNV_SUBSCRIPTION_SOURCE}
+  source: quay.io/openshift-cnv/nightly-catalog
```

Comment:
previously this was `cnv-prerelease-catalog-source` but now it's `quay.io/openshift-cnv/nightly-catalog` ?